rustler_codegen/src/tuple.rs

```diff
@@ -34,10 +34,15 @@ pub fn transcoder_decorator(ast: &syn::DeriveInput) -> TokenStream {
 }
 
 fn gen_decoder(ctx: &Context, fields: &[&Field]) -> TokenStream {
-    let struct_type = &ctx.ident_with_lifetime;
     let struct_name = ctx.ident;
     let struct_name_str = struct_name.to_string();
 
+    let lifetimes = &ctx.lifetimes;
+    // FIXME maybe put in context
+    let mut rustler_decoder_lifetimes = vec![];
+    rustler_decoder_lifetimes.resize(lifetimes.len(), quote! { 'a });
```

Comment:
Why resize?

Reply:
I didn't know a cleverer way to create a vector of length `lifetimes.len()` containing `'a`.

Reply:
`vec![quote! { 'a }, lifetimes.len()]`